spec/inputs/rabbitmq_spec.rb

```diff
@@ -54,15 +60,42 @@
       subject { hare_info }
 
       it "should set the queue correctly" do
+        register_instance
```

Comment:
Basically because it wouldn't do the right thing for a more inner test that comes up later. For the reconnection case we have to call `setup_mocks` directly.

While the tests could be refactored to obviate this, I think being a little more verbose isn't bad enough to justify the time to refactor out the tests even more.